assets/js/modules/analytics-4/components/setup/SetupEnhancedMeasurementSwitch.test.js

```diff
@@ -332,37 +285,6 @@ describe( 'SetupEnhancedMeasurementSwitch', () => {
 				);
 		} );
 
-		it( 'should revert the switch from off to on', () => {
-			registry
```

Comment:
I movd these to setup form tests, as logic is changed for EM switch to be reset when user changes the dropdown, instead of whenever values change in datastore, which made it impossible to disable EM